### PR TITLE
HDDS-5736. Update navbar from Hadoop to Ozone on doc page

### PR DESCRIPTION
**Pull request by sky76093016**
## What changes were proposed in this pull request?

Update navbar from Hadoop to Ozone on doc page.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-5736

## How was this patch tested?

Local hugo test.(as shown below)
image

### CODE REVIEW

**Comment by sky76093016** [comment]
@cxorm @cku328 Please take a look.

File: hadoop-hdds/docs/themes/ozonedoc/layouts/partials/navbar.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
     <div id="navbar" class="navbar-collapse collapse">
       <ul class="nav navbar-nav navbar-right">
         <li><a href="https://github.com/apache/hadoop-ozone">Source</a></li>
```
Comment (ayushtkn) [inline hadoop-hdds/docs/themes/ozonedoc/layouts/partials/navbar.html:36]: Should change to 
https://github.com/apache/ozone

**Review by ayushtkn — COMMENTED** [review]
Can you give a check, I found similar stuff in sidebar.html as well.

There are couple of entries in README.md also pointing to hadoop doc, give a check if they can be updated as well

File: hadoop-hdds/docs/themes/ozonedoc/layouts/partials/navbar.html (hunk 1 above)
Comment (sky76093016) [reply hadoop-hdds/docs/themes/ozonedoc/layouts/partials/navbar.html:36]: Thanks to @ayushtkn  for the review.
Of course, let me confirm what you mentioned.

Comment (sky76093016) [reply hadoop-hdds/docs/themes/ozonedoc/layouts/partials/navbar.html]: @ayushtkn I searched the README.md under Ozone, there should be no other errors

**Review by cku328 — APPROVED** [review]
Overall LGTM. Thanks @sky76093016 for the work.

**Review by bharatviswa504 — APPROVED** [review]

**Comment by bharatviswa504** [comment]
Thank You @sky76093016 for the contribution @cku328 and @ayushtkn for the review

**Pull request merged**